### PR TITLE
adding materials for ACCESS request

### PR DESCRIPTION
**Pull request by AstrobioMike**
@ctb can you also add me as a collaborator to the stamps2023 repo, por favor?

### CODE REVIEW

**Comment by ctb** [comment]
@AstrobioMike you're an admin on this repo already...?

File: ACCESS-jetstream2-computing-resource-request/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+**Fields of Science:** “Other Biological Sciences”
+
+**Documents and Publications:** Needed to attach my CV here.
+
+---
+
+## After submission
+
+### Transfering credits from ACCESS to JetStream2
```
Comment (ctb) [inline ACCESS-jetstream2-computing-resource-request/README.md]:
```suggestion
### Transferring credits from ACCESS to JetStream2
```

File: ACCESS-jetstream2-computing-resource-request/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Once approved, and logged in, needed this page (https://allocations.access-ci.org/requests) in order to transfer ACCESS credits to specific resource as detailed on this page (https://allocations.access-ci.org/use-credits-overview). For the appropriate allocation request, selected "Choose New Action", then "Exchange". On next screen, chose Indiana Jetstream2 as Resource, clicked "Add Resource", entered all credits, needed to add a comment, and then submitted.
```
Comment (ctb) [inline ACCESS-jetstream2-computing-resource-request/README.md]:
```suggestion
Once approved, and logged in, use this page (https://allocations.access-ci.org/requests) in order to transfer ACCESS credits to the specific resource as detailed on this page (https://allocations.access-ci.org/use-credits-overview). For the appropriate allocation request, selected "Choose New Action", then "Exchange". On next screen, chose Indiana Jetstream2 as Resource, clicked "Add Resource", entered all credits, needed to add a comment, and then submitted.
```

**Review by ctb — APPROVED** [review]
go for it! see two suggestions.

**Comment by AstrobioMike** [comment]
> @AstrobioMike you're an admin on this repo already...?

i thought so, oh i guess i couldn't merge due to review being needed now?

**Comment by ctb** [comment]
huh, no, I don't have any branch protection rules or anything. weird! will debug if it happens next time.

**Pull request merged**